.github/workflows/promote.yaml

```diff
@@ -8,13 +8,15 @@ on:
         description: Channel to promote to
         options:
           - beta
+          - candidate
           - stable
       promote-from:
         type: choice
         description: Channel to promote from
         options:
-          - beta
           - edge
+          - beta
+          - candidate
```

Comment:
Seems like it would be possible to "promote" from stable to candidate.
Is that intended?
Should we instead have only one input - "Channel to promote"?

Reply:
I'm not sure I understand. Stable is not a choice for "promote-from". I don't see how we could have only one.

Reply:
Okay I see what you're saying here. I think there could be times where we promote up two levels. I think the flexibility would be nice. What do others think?